features/cellular/framework/API/CellularNetwork.h

```diff
@@ -107,7 +108,8 @@ class CellularNetwork {
         RAT_E_UTRAN,
         RAT_CATM1,
         RAT_NB1,
-        RAT_UNKNOWN
+        RAT_UNKNOWN,
+        RAT_MAX = 11 // to reserve string array
```

Comment:
adding new enum should be a separate commit (separate from "debug prints update")